src/styles/index.scss

```
@@ -1,3 +1,110 @@
+@import './variables';
+@import './stars';
+
+* {
+  padding: 0;
+  font-family: $font-main;
+}
+
+html {
+  font-size: $font-size-base;
+}
+
 body {
   margin: 0;
 }
+
+.product-card {
+  border: 1px solid;
+  border-color: #f3f3f3;
+  width: 198px;
+  height: 406px;
+  border-radius: 5px;
+  margin-top: 1px;
+  margin-left: 1px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+    margin-left: 19px;
+    margin-bottom: 40px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    font-family: $font-main;
+    margin-left: 16px;
+    margin-right: 16px;
+  }
+
+  &__code {
+    font-family: $font-main;
```

The product code text is not black. Its color is #616070.
```suggestion
  &__code {
    color: #616070;
    font-family: $font-main;
```